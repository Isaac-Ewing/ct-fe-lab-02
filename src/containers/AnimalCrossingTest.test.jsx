import React from 'react';
import { render, screen } from '@testing-library/react';
import AnimalCrossingContainer from './AnimalCrossing';

describe('Animal Crossing Container', () => {
  it('renders a list of villagers', async () => {
    render(<AnimalCrossingContainer />);
    const ul = await screen.findByRole('list', { name: 'characters' });

    expect(ul).toMatchSnapshot();
    
  });
});
