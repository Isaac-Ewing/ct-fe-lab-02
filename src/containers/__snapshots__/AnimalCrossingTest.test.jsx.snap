// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Animal Crossing Container renders a list of villagers 1`] = `
<ul
  aria-label="characters"
/>
`;
